docs/release/fundamentals/tables-and-data-operations.ipynb

```diff
@@ -0,0 +1,2565 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "GwjjornhycHU"
+   },
+   "outputs": [],
+   "source": [
+    "%pip install -q pixeltable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Tables\n",
+    "\n",
+    "All data in Pixeltable is stored in tables. At a high level, a Pixeltable table behaves similarly to an ordinary SQL database table, but with many additional capabilities to support complex AI workflows. We'll introduce those advanced capabilities gradually throughout this tutorial; in this section, the focus is on basic table and data operations.\n",
+    "\n",
+    "Tables in Pixeltable are grouped into __directories__, which are simply user-defined namespaces. The following command creates a new directory, `fundamentals`, which we'll use to store the tables in our tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "EIBhrFB30cbF",
+    "outputId": "9c30c33d-e16b-4aeb-cfc5-9a5770416d59"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connected to Pixeltable database at: postgresql://postgres:@/pixeltable?host=/Users/asiegel/.pixeltable/pgdata\n",
+      "Created directory `fundamentals`.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<pixeltable.catalog.dir.Dir at 0x33f0b2040>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pixeltable as pxt\n",
+    "\n",
+    "# First we delete the `fundamentals` directory and all its contents (if\n",
+    "# it exists), in order to ensure a clean environment for the tutorial.\n",
+    "pxt.drop_dir('fundamentals', force=True)\n",
+    "\n",
+    "# Now we create the directory.\n",
+    "pxt.create_dir('fundamentals')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "7StQcO-i-7NF"
+   },
+   "source": [
+    "Now let's create our first table. To create a table, we must give it a name and a __schema__ that describes the table structure. Note that prefacing the name with `fundamentals` causes it to be placed in our newly-created directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Created table `films`.\n"
+     ]
+    }
+   ],
+   "source": [
+    "films_t = pxt.create_table('fundamentals.films', {\n",
+    "    'film_name': pxt.StringType(),\n",
+    "    'year': pxt.IntType(),\n",
+    "    'revenue': pxt.FloatType()\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To insert data into a table, we use the `insert()` method, passing it a list of Python dicts."
```

Comment:
Probably a good place to mention again that there's also update() and delete(), and you'll get to those later.